core-developers/experts.rst

```diff
@@ -282,14 +282,20 @@ PEG Generator       gvanrossum, pablogsal, lysnikolaou
 ==================  ===========
 
 
+.. _platform-experts:
+
 Platforms
 =========
 
+For official contacts for supported platforms, see :pep:`11`.
+
+Platforms listed here are not necessarily supported by CPython, and
+the maintainers are not necessarily core CPython developers.
```

Comment:
> ... the maintainers are not necessarily core CPython developers.

This echoes the first sentence of this page:

> ... names for each item that indicate a maintainer or an expert in the field.

But fine to keep if you wish to re-emphasise it here?

Reply:
I do wish to re-emphasize, because this is more prominent in this table. Core devs listed here should ideally be PEP-11 contacts, and there's not much use in repeating them here. (That is, I don't think this is a good place for macOS and Windows, but removing them isn't worth the churn at this point.)

But maybe it's not important to say that here. My latest commit adds this instead:

> Platforms listed here are not necessarily supported by CPython.
Some of the experts listed here maintain and distribute Python
for “their” platform as a third-party project.

IMO that's a more interesting fact :)